functions/passing-parameters-by-value.md

```diff
@@ -0,0 +1,91 @@
+# Module name: Passing Parameters by Value
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+## Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational</td>
+    <td>Defining and calling functions with values</td>
+  </tr>
+  <tr>
+    <td>Main</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td></td>
+  </tr>
+</table>
+
+## Motivation
+
+_Why is this important?_
+_Why do we want to learn/teach this topic?_
+
+The fundamental element for code-reuse is a function. For functions to be 
```

Comment:
I would say "code modularization". Many functions are only called once, so there is often no reuse.